docs/modules/ROOT/pages/development.adoc

```diff
@@ -440,7 +440,7 @@ class MyCop < Base
   requires_gem "my-gem"
 
   def on_send(node)
-    if target_gem_version("my-gem) < "1.0"
+    if target_gem_version("my-gem) < "2.0"
```

Comment:
No strong reason to change this, it just works better with the examples already given, and the one I now added